### PR TITLE
Issue 2851 upload terminology valueset parallel versioning

### PR DESCRIPTION
**Pull request by jmarchionatto**

### CODE REVIEW

**Comment by jmarchionatto** [comment]
Change to improve code readability. Changed a utility method which name was not expressing intention clearly.

File: hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/term/BaseTermReadSvcImpl.java [hunk 1]
```diff
@@ -2349,12 +2348,10 @@ public CodeValidationResult codeSystemValidateCode(IIdType theCodeSystemId, Stri
 	@Override
 	public Optional<TermValueSet> findCurrentTermValueSet(String theUrl) {
 		if (TermReadSvcUtil.isLoincNotGenericUnversionedValueSet(theUrl)) {
-			if (TermReadSvcUtil.mustReturnEmptyValueSet(theUrl))   return Optional.empty();
+			Optional<String> vsIdOpt = TermReadSvcUtil.getValueSetId(theUrl);
+			if (! vsIdOpt.isPresent())  return Optional.empty();
```
Comment (tadgh) [inline hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/term/BaseTermReadSvcImpl.java:2352]: This isn't strictly enforced, but using implicit braces is really not done anywhere else in the HAPI-FHIR repo. Eventually, I would like style stuff like this to happen automatically via commit hook, but for now we're stuck enforcing in code-review land. Could you change this to: 
```
if (!vsIdOpt.isPresent()) {
    return Optional.empty();
}
```

Comment (tadgh) [reply hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/term/BaseTermReadSvcImpl.java:2352]: not a blocker for merging though

**Review by tadgh — APPROVED** [review]

File: hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/term/BaseTermReadSvcImpl.java (hunk 1 above)
Comment (jmarchionatto) [reply hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/term/BaseTermReadSvcImpl.java]: Done. However I could argue for readability improvement using implicit brackets in these specific use cases (one liner return)

**Pull request merged**